src/status_im/ui/components/list_item/views.cljs

```diff
@@ -38,15 +39,26 @@
                       :ellipsize-mode  :tail}
           subtitle])]
       ;;Accessories
+      [react/view {:flex 1}]
       (for [accessory accessories]
         (with-meta
           (cond
-            (string? accessory)
-            [react/text {:style styles/accessory-text}
-             accessory]
             (= :chevron accessory)
-            [icons/icon :main-icons/next {:color colors/gray-transparent-40}]
+            [react/view
+             [icons/icon :main-icons/next {:color colors/gray-transparent-40}]]
             (= :check accessory)
-            [icons/icon :main-icons/check {:color colors/gray}]
-            :else accessory)
-          {:key accessory}))]]))
+            [react/view
+             [icons/icon :main-icons/check {:color colors/gray}]]
+            :else
+            [react/view {:padding-right 8 :flex-shrink 1}
```

Comment:
I disagree with this change, it is adding an unecessary wrapper around all the components below, and this has a performance impact. The components below can support this styling themselves

Reply:
this is something new from performance investigation, i can fix this here for sure, but we need general performance guidelines

Reply:
in this case it's simple that can kind of over wrapping is an extra thing to render on native-side, and since it is in the list it's an extra wrapping for each element. And it is only used for some styling it is unecessary because all elements support that kind of styling